### PR TITLE
Update Breadcrumbs and CodeEditor styles

### LINKED ISSUE #936: Tweak code editor bar look-n-feel
### Mockup

codebar1-2

The common breadcrumb mechanism is used to denote C++ mode

### Icons

[back.zip](https://github.com/xodio/xod/files/1538285/back.zip)

### Acceptance criteria

- [ ] The same breadcrumbs as in the debugger are used
- [x] The bar and its elements are colored as on the mockup
- [x] Code editor background color matches the one in the mockup

### Out of scope

- Syntax highlight color adjustments
- Insert/Verify buttons

### PR DESCRIPTION
**Pull request by brusherru**
it closes #936.

### CODE REVIEW

File: packages/xod-client/src/core/styles/components/CodeMirror.scss [hunk 1]
```diff
@@ -402,3 +402,57 @@ span.CodeMirror-selectedtext { background: none; }
...
+    Author:     XOD (http://github.com/xodio/xod)
+
+*/
+
+.cm-s-xod.CodeMirror {
+  background-color: $blackberry-shadow;
+  color: $chalk-bright;
+  font-family: $font-family-mono;
+  font-size: 12px;
+  font-weight: 300;
```
Comment (nkrkv) [inline packages/xod-client/src/core/styles/components/CodeMirror.scss]: On my machine 300 is hard to read. 400 works much better

File: packages/xod-client/src/core/styles/components/CodeMirror.scss [hunk 2]
```diff
@@ -402,3 +402,57 @@ span.CodeMirror-selectedtext { background: none; }
...
+.cm-s-xod .cm-variable-2 { color: $apricot-light; }
+.cm-s-xod .cm-variable-3, .cm-s-xod .cm-type { color: $apricot-light; }
+.cm-s-xod .cm-builtin { color: $apricot-light; }
+.cm-s-xod .cm-atom { color: $red-bright; }
+.cm-s-xod .cm-number { color: $red-bright; }
+.cm-s-xod .cm-def { color: $chalk; }
+.cm-s-xod .cm-string { color: $apricot-light; }
+.cm-s-xod .cm-string-2 { color: $apricot-light; }
+.cm-s-xod .cm-comment { color: $light-grey-bright; }
+.cm-s-xod .cm-variable { color: #77A0FF; }
```
Comment (nkrkv) [inline packages/xod-client/src/core/styles/components/CodeMirror.scss]: Simple `$chalk` works better for `cm-variable`

File: packages/xod-client/src/core/styles/components/CodeMirror.scss [hunk 3]
```diff
@@ -402,3 +402,57 @@ span.CodeMirror-selectedtext { background: none; }
...
+
+.cm-s-xod .CodeMirror-activeline-background { background: rgba(0, 0, 0, 0); }
+.cm-s-xod .cm-keyword { color: #E67CDC; }
+.cm-s-xod .cm-operator { color: $chalk-bright; }
+.cm-s-xod .cm-variable-2 { color: $apricot-light; }
+.cm-s-xod .cm-variable-3, .cm-s-xod .cm-type { color: $apricot-light; }
+.cm-s-xod .cm-builtin { color: $apricot-light; }
+.cm-s-xod .cm-atom { color: $red-bright; }
+.cm-s-xod .cm-number { color: $red-bright; }
+.cm-s-xod .cm-def { color: $chalk; }
```
Comment (nkrkv) [inline packages/xod-client/src/core/styles/components/CodeMirror.scss]: `#77A0FF` works better for `cm-def`

**Comment by brusherru** [comment]
@nkrkv fixed

**Pull request merged**